pkgs/by-name/to/toml-sort/package.nix

```
@@ -0,0 +1,32 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+}:
+let
+  version = "0.23.1";
+in
+python3Packages.buildPythonPackage {
+  pname = "toml-sort";
+  inherit version;
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "pappasam";
+    repo = "toml-sort";
+    rev = "v${version}";
```

```suggestion
    rev = "refs/tags/v${version}";
```
Avoid tags/branch confusion